OpenDirectoryDownloader/OpenDirectoryIndexer.cs

```diff
@@ -480,14 +481,29 @@ public async void StartIndexingAsync()
                     Console.WriteLine("Finished indexing!");
 
                     Program.SetConsoleTitle($"✔ {Program.ConsoleTitle}");
-
+                    
+                    Boolean clipboardSuccess = false;
```

Comment:
Please change this tool "bool" and add a empty line after it. I thought it was possible to do single line changes in PR, can't find how 😮

Reply:
will do!

Reply:
I actually had a blank line in-between at first but though it'd be good to group things together xD

Reply:
Haha. Yes, I understand. I like some space 🤣